src/__experimental__/components/form/form.component.js

```diff
@@ -221,23 +222,20 @@ class FormWithoutValidations extends React.Component {
   }
 
   async submitControlledForm() {
-    const response = await fetch(
-      this.props.formAction, {
-        headers: {
-          Accept: 'application/json',
-          'Content-Type': 'application/json'
-        },
-        method: 'post',
-        body: JSON.stringify(this.state.formInputs)
-      }
-    );
-    return this.clearFormData(await response.json());
+    const response = await axios.post(this.props.formAction, {
+      ...this.state.formInputs
+    }, {
+      Accept: 'application/json',
+      'Content-Type': 'application/json'
+    });
```

Comment:
there is an util to handle ajax calls, shouldn't it be used here?
https://github.com/Sage/carbon/blob/master/src/utils/service/service.js

Reply:
Yeah good point, I've refactored it to use the service